week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -5,7 +5,24 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+function tidyUpString() {
+  let strArr = ["/Daniel ", "irina ", " Gordon", "ashleigh "];
+  for (var i = 0; i < strArr.length; i++) {
+    strArr[i] = strArr[i].trim();
+    strArr[i] = strArr[i].replace(/\//g, "");
+    strArr[i] = strArr[i].toLowerCase();
+  }
+  return strArr;
+}
+function tidyUpString1() {
+  let strArr = [" /Sanyia ", " Michael ", "AnTHonY ", "   Tim   "];
+  for (var i = 0; i < strArr.length; i++) {
+    strArr[i] = strArr[i].trim();
+    strArr[i] = strArr[i].replace(/\//g, "");
+    strArr[i] = strArr[i].toLowerCase();
+  }
+  return strArr;
+}
```

Comment:
Notice that, similarly to one of the previous functions, `tidyUpString1()` is called in the test below with a parameter that contain the list of strings so you shouldn't recreate it inside of the function.

Other than that the function looks good 👍

Btw, I guess you created that new function to try something but don't forget to remove code like this once you're done with it :)